ui/warlock/sim.ts

```diff
@@ -18,44 +18,30 @@ const SPEC_CONFIG = registerSpecConfig(Spec.SpecWarlock, {
 	// All stats for which EP should be calculated.
 	epStats: [
 		// Primary
-		Stat.StatHealth,
 		Stat.StatMana,
 		// Attributes
-		Stat.StatStrength,
-		Stat.StatStamina,
 		Stat.StatIntellect,
 		Stat.StatSpirit,
-		Stat.StatAgility,
-		// Physical
-		Stat.StatAttackPower,
-		Stat.StatMeleeCrit,
-		Stat.StatMeleeHit,
 		// Spell
 		Stat.StatSpellPower,
 		Stat.StatSpellDamage,
 		Stat.StatSpellHit,
 		Stat.StatSpellCrit,
-		Stat.StatSpellHaste,
```

Comment:
"Haste" can stay in I believe as other classes also show it? But iirc this is not even a classic stat?

Reply:
I don't think we actually show it in the "Classic" codebase looking around at other sims. In SoD we use the cast speed multiplier in place of `StatSpellHaste` since that stat is effectively ueless when summed up, but in vanilla there wasn't any gear with cast speed modifiers to my knowledge